frontends/open-discussions/scss/breakpoint.scss

```diff
@@ -0,0 +1,27 @@
+@mixin breakpoint($name) {
```

Comment:
This is really a file move, not a new file.

In this PR:
- `infinite-corridor/src/scss/breakpoints.scss` became new `ol-util/src/scss/breakpoint.scss`
- old `ol-util/src/scss/breakpoint.scss`, which was not used in new codebase, became this file.